src/Groups/directproducts.jl

```diff
@@ -472,10 +472,10 @@ end
 
 function Base.show(io::IO, x::SemidirectProductGroup)
    if x.isfull
-      print(io, "SemidirectProduct( ", String(GAP.Globals.StringViewObj(x.N.X)),
+      print(io, "SemidirectProduct( ", String(GAPWrap.StringViewObj(x.N.X)),
```

Comment:
@CameronBraunstein can you please add examples to the `semidirect_product` docstring? Then this code here will automatically get coverage, plus I think any user of `semidirect_product` will appreciate it.

I guess the same holds for other related functions, such as `projection` and `embedding`